### PR TITLE
added api docs for user management

### PR DESCRIPTION
**Pull request by mmacata**
APIs to
-  list all users, to 
- create a user, to 
- show detailed information about a user including group, role and permissions and to
- delete a user already existed.
In the api docs they did not appear. With this PR they do.

### CODE REVIEW

File: src/actinia_core/resources/common/response_models.py [hunk 1]
```diff
@@ -1150,3 +1150,124 @@ def create_response_from_model(response_model_class=ProcessingResponseModel,
...
+            'items': {'type': 'string'},
+            'description': 'The names of all users'
+        }
+    }
+    required = ["status", "user_list"]
+    example = {"status": "success","user_list": ["actinia-gdi"]}
+
+
+
+class UserListResponseModel(Schema):
```
Comment (neteler) [inline src/actinia_core/resources/common/response_models.py]:
```suggestion
class UserListResponseModel(Schema):
```

Is this class a dupe of line 1155 etc?

File: src/actinia_core/resources/common/response_models.py [hunk 2]
```diff
@@ -1150,3 +1150,124 @@ def create_response_from_model(response_model_class=ProcessingResponseModel,
...
+        'status': {
+            'type': 'string',
+            'description': 'The status of the request'
+        },
+        'permissions': {
+            'type': 'object',
+            'properties': {
+                'cell_limit': {
+                    'type': 'string',
+                    'description': 'The limit of number of cells the user is allowed to process'
```
Comment (neteler) [inline src/actinia_core/resources/common/response_models.py]:
```suggestion
                    'description': 'The limit of number of raster cells the user is allowed to process'
```

File: src/actinia_core/resources/common/response_models.py [hunk 3]
```diff
@@ -1150,3 +1150,124 @@ def create_response_from_model(response_model_class=ProcessingResponseModel,
...
+                    'description': 'The limit of number of processes the user is allowed to integrate into one process chain'
+                },
+                'process_time_limit': {
+                    'type': 'string',
+                    'description': 'The time a process must not exceed'
+                },
+                'accessible_datasets': {
+                    'type': 'object',
+                    'properties': {},
+                    'description': 'The persistent GRASS databases the user is allowed to use. Contains one object for each location name with an array of strings containing all allowed mapset names. See example for more information.'
```
Comment (neteler) [inline src/actinia_core/resources/common/response_models.py]:
```suggestion
                    'description': 'The persistent GRASS GIS database(s) the user is allowed to use. Contains one object for each location name with an array of strings containing all allowed mapset names. See example for more information.'
```

File: src/actinia_core/resources/common/response_models.py [hunk 4]
```diff
@@ -1150,3 +1150,124 @@ def create_response_from_model(response_model_class=ProcessingResponseModel,
...
+                },
+                'accessible_modules': {
+                    'type': 'array',
+                    'items': {'type': 'string'},
+                    'description': 'The GRASS modules the user is allowed to use'
```
Comment (neteler) [inline src/actinia_core/resources/common/response_models.py]:
```suggestion
                    'description': 'The GRASS GIS modules the user is allowed to use'
```

File: src/actinia_core/resources/common/response_models.py (hunk 1 above)
Comment (mmacata) [reply src/actinia_core/resources/common/response_models.py]: ups, I fixed it

**Comment by mmacata** [comment]
Changed it, pls review again @neteler

**Comment by mmacata** [comment]
Then I'll merge now

**Pull request merged**